app/event-publisher/index.js

```diff
@@ -94,3 +97,53 @@ export const raiseClaimEvents = async (event, sbi = 'none') => {
     }
   ])
 }
+
+export const raiseApplicationFlaggedEvent = async (event, sbi) => {
+  await new PublishEventBatch(config.eventQueue).sendEvents([
+    {
+      name: SEND_SESSION_EVENT,
+      properties: {
+        id: event.application.id,
+        sbi,
+        cph: 'n/a',
+        checkpoint: process.env.APPINSIGHTS_CLOUDROLE,
+        status: 'success',
+        action: {
+          type: 'application:flagged',
+          message: event.message,
+          data: {
+            flagId: event.flag.id,
+            flagDetail: event.flag.note,
+            flagAppliesToMh: event.flag.appliesToMh
+          },
+          raisedBy: event.raisedBy,
+          raisedOn: event.raisedOn.toISOString()
+        }
+      }
+    }
+  ])
+}
+
+export const raiseApplicationFlagDeletedEvent = async (event, sbi) => {
+  await new PublishEventBatch(config.eventQueue).sendEvents([
+    {
+      name: SEND_SESSION_EVENT,
+      properties: {
+        id: event.application.id,
+        sbi,
+        cph: 'n/a',
+        checkpoint: process.env.APPINSIGHTS_CLOUDROLE,
+        status: 'success',
+        action: {
+          type: 'application:unflagged',
+          message: event.message,
+          data: {
+            flagId: event.flag.id
```

Comment:
Has a ticket been raised to add the deletedNote into here? (As well as everywhere else that needs it, to accept, persist, return etc)

Reply:
Indy has created a separate ticket for all of the deletedNote stuff yeah

Reply:
https://eaflood.atlassian.net/browse/AHWR-720